lunar/distribution.yaml

```diff
@@ -1728,6 +1728,22 @@ repositories:
       url: https://github.com/ros-drivers/rgbd_launch.git
       version: jade-devel
     status: maintained
+  robot_localization:
+    doc:
+      type: git
+      url: https://github.com/cra-ros-pkg/robot_localization.git
+      version: lunar-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/cra-ros-pkg/robot_localization-release.git
+      version: 2.4.0-1
+    source:
+      test_pull_requests: true
```

Comment:
Have you granted access to the @ros-pull-request-builder user for your org unit or setup the webhooks manually and provide access to the repo?